src/app/components/memberInfoForm/memberInfoFormContainer.js

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+
+import { deleteMember } from '../../actions/teamActions';
+
+import MemberInfoForm from './memberInfoFormComponent';
```

Comment:
It's generally not a good idea to have nested components that are both containers connected to Redux. In this case, I think all of the props that MemberInfoForm needs can be passed in from the add & edit parent components.